### PR TITLE
[automl image] Add training in batch scoring notebook and reduce GPU usage in all notebooks

### PR DESCRIPTION
**Pull request by martinezgjuan**
# PR into Azure/azureml-examples

## Checklist

I have:

- [X] read and followed the contributing guidelines

## Changes

- Add training in batch scoring notebook and reduce GPU usage in all notebooks

### CODE REVIEW

**Review by jeff-shepherd — APPROVED** [review]

File: python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "# Batch Predictions for an Image Classification model trained using AutoML\n",
     "In this notebook, we go over how you can use [Azure Machine Learning pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-pipeline-batch-scoring-classification) to run a batch scoring image classification job.\n",
     "\n",
-    "**Please note:** Before running this notebook, you need to train an image classification model using AutoML for Images. Refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) to know how to train an image classification model with AutoML. "
+    "**Please note:** Before running this notebook, you need to train an image classification model using AutoML for Images. Refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) or to the [image classification multiclass notebook](https://github.com/Azure/azureml-examples/blob/main/python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb) to know how to train an image classification model with AutoML. "
```
Comment (raduk) [inline python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb:14]: this is not true anymore - we should add something along the lines "we include below a sample training for convenience and to be able to run the notebook end to end "

File: python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb [hunk 2]
```diff
@@ -441,8 +441,8 @@
     "}\n",
     "\n",
     "tuning_settings = {\n",
-    "    \"iterations\": 20,\n",
-    "    \"max_concurrent_iterations\": 4,\n",
+    "    \"iterations\": 10,\n",
```
Comment (raduk) [inline python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb:444]: please adapt the comment as well: "We use Random Sampling to pick samples from this parameter space and try a total of 20 iterations with these different samples, running 4 iterations at a time on our compute target, which has been previously set up using 4 nodes. Please note that the more parameters the space has, the more iterations you need to find optimal models."

File: python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb (hunk 1 above)
Comment (martinezgjuan) [reply python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb:14]: How does this sound?

**Please note:** For this notebook you can use an existing image classification model trained using AutoML for images or use the simple model training we included below for convenience. For detailed instructions on how to train an image classification model with AutoML, please refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) and to the [image classification multiclass notebook](https://github.com/Azure/azureml-examples/blob/main/python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb).

Comment (martinezgjuan) [reply python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb]: Updated. Please check.

**Review by jeff-shepherd — APPROVED** [review]
I like the idle_seconds_before_scaledown change.

**Review by raduk — APPROVED** [review]

**Pull request merged**